doc/release/2.4.2.rst

```diff
@@ -31,14 +31,15 @@ Functionality added or changed
 Core
 ~~~~
 
--   ``box.session.push()`` parameter ``sync`` is deprecated. A warning is
-    printed when the sync is used, but it still works. It is removed in
-    the next version (gh-4689).
+-   :doc:`</reference/reference_lua/box_session/push>` parameter ``sync``
```

Comment:
```suggestion
-   The ``sync`` parameter of the :doc:`</reference/reference_lua/box_session/push>` function 
```

Reply:
Not sure about that: if we are going to proof-read and correct the RN items for the previous release in our documentation.

Reply:
I think we should put it in a separate issue